### ISSUE TITLE
Swing animation not in-sync with hit when using COMBAT_ANIM_HIT_SMOOTH

### ISSUE THREAD

**Issue by a1exp** [Sphereserver/Source-X#788]
I'm using `COMBAT_ANIM_HIT_SMOOTH` as it's the only way to avoid weird dealy before the first hit when you start combat, but there is another issues players are complaing. 

Animation of swing does not correspond to actual swing time. So you see that animation finished and only after some delay the actual hit is done and sound of hit played. People who play without sound don't actually understand now if the hit was done or not since they are relying on animation like it was on Sphere 56b. 

Is there a way to make that animation ends when the hit is actually performed when `COMBAT_ANIM_HIT_SMOOTH` is used?

https://user-images.githubusercontent.com/31130012/138567826-f6b0c0b7-1ba6-4a19-a88e-424b4f8114cb.mp4

**Comment by canerksk** [comment]
This experimental project has been ongoing since its inception. It has a method with script, but it was not working stable either.

**Comment by wagneruo** [comment]
ooh, old issue and still here :(

**Comment by drk84** [comment]
Close because a duplicate of Issue #1011
